README.md

```diff
@@ -27,61 +27,80 @@ npm install --save-dev @types/i18next-fs-backend
 
 ### Configuration
 
-Then create a `i18n.server.ts` file somewhere in your app and add the following code:
+First, set your [i18next configuration](https://www.i18next.com/overview/configuration-options).
+
+These two files can go somewhere in your app folder.
+
+For this example, we will create `app/i18n.ts`:
 
 ```ts
-import Backend from "i18next-fs-backend";
-import { resolve } from "node:path";
-import { RemixI18Next } from "remix-i18next";
+export default {
+    // This is the list of languages your application supports
+    supportedLngs: ['en', 'es'],
+    // This is the language you want to use in case
+    // if the browser language is not in the supportedLngs
```

Comment:
```suggestion
    // if the user language is not in the supportedLngs
```
The language could also come from a DB so it may be the user preferred language and not the browser one.